src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -162,6 +186,14 @@ const AppLayout: FC = ({ children }) => {
 		materialRequests && dispatch(setMaterialRequestCount(materialRequests?.items.length));
 	}, [dispatch, materialRequests]);
 
+	useEffect(() => {
+		if (!isLoggedIn) {
+			return;
+		}
+
+		getVisitorSpaces();
```

Comment:
just for clarity:
visit space is not the same thing as a visit request

one is a representation of a physical room that someone can visit
and the other is a request to visit the room at a certain date

So this function would be called: getActiveVisitRequests()

But like stated below => its better to use the hook: useGetAllActiveVisits